org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/angular/AngularLanguageServer.java

```diff
@@ -37,8 +50,12 @@ public AngularLanguageServer() {
 			commands.add("--tsProbeLocations");
 			commands.add(new File(nodeModules, "typescript").getAbsolutePath());
 			commands.add("--stdio");
-			commands.add("--logFile");
-			commands.add(Platform.getLogFileLocation().removeLastSegments(1).append("angular-language-server-" + System.currentTimeMillis() + ".log").toFile().getAbsolutePath());
+			if (isLoggingToFileEnabled) {
+				commands.add("--logFile");
+				commands.add(Platform.getLogFileLocation().removeLastSegments(1)
+					.append("angular-language-server.log").toFile()
+					.getAbsolutePath());
+			}
 			commands.add("--logVerbosity");
 			commands.add("terse");
```

Comment:
By the way, can we switch this to `off` when no logging is required and `verbose` when logging is required?

Reply:
According to this comment, there is no 'off' option for logVerbosity. cf. https://github.com/angular/vscode-ng-language-service/pull/584#issuecomment-578983768

I think we should merge this PR and create new one when angular language server will be updated with an `off` option.

Reply:
Ok, please rebase and merge it.